chemclipse/plugins/org.eclipse.chemclipse.chromatogram.msd.peak.detector/src/org/eclipse/chemclipse/chromatogram/msd/peak/detector/settings/IPeakDetectorSettingsMSD.java

```diff
@@ -11,7 +11,25 @@
  *******************************************************************************/
 package org.eclipse.chemclipse.chromatogram.msd.peak.detector.settings;
 
+import java.util.Collection;
+
 import org.eclipse.chemclipse.chromatogram.peak.detector.settings.IPeakDetectorSettings;
+import org.eclipse.chemclipse.numeric.statistics.WindowSize;
 
 public interface IPeakDetectorSettingsMSD extends IPeakDetectorSettings {
+
+	enum FilterMode {
+		INCLUDE, EXCLUDE
```

Comment:
what if the user wants to neither include nor exclude, but want the old behavior?

Reply:
The old behavior is EXCLUDE with an empty List.